multimetric-demo/svm_example.ipynb

```diff
@@ -199,7 +199,10 @@
     "svm_score_time_meta = {\n",
     "    'name': 'SigOpt Multimetric Demo - Balancing Score vs. Training Time (python)',\n",
     "    'project': 'sigopt-examples',\n",
-    "    'metrics': [{'name': 'score'}, {'name': 'training time'}],\n",
+    "    'metrics': [\n",
+    "        {'name': 'score'},\n",
```

Comment:
I think these are missing the objectives

Reply:
also since `training time` is negated, will have to update the observation values to not be negative

Reply:
also there's another mention of negating here `https://github.com/sigopt/sigopt-examples/pull/127/files#diff-3c62cc8701b9e60c3e2fd912f2421925R181`